spec/jobs/affiliate_indexed_document_fetcher_job_spec.rb

```diff
@@ -61,9 +61,42 @@
     end
   end
 
-  context 'when affiliate or indexed document have disappeared before job runs' do
+  context 'when an indexed document has disappeared before job runs' do
+    let(:unfetched2) { IndexedDocument.find_by(unfetched_atts2) }
+    let(:unfetched_atts2) do
+      { url: 'http://nps.gov/foo2.html',
+        title: 'Doc Title 2',
+        description: 'This is a document 2.' }
+    end
+
+    before do
+      affiliate.indexed_documents.build(unfetched_atts2)
+      affiliate.save
```

Comment:
I recommend simplifying these two lines as:
```ruby
affiliate.indexed_documents.create!(unfetched_atts2)
```
I prefer to use the bang methods in specs to ensure that an error will be raised if the spec setup is faulty. However, this is just a suggestion, so I'll go ahead and approve this PR.

Reply:
Good call.  Change made.